server/graphql/v2/interface/IsMemberOf.js

```diff
@@ -151,6 +161,36 @@ export const IsMemberOfFields = {
             as: 'collective',
             where: collectiveConditions,
             required: true,
+            attributes: {
+              include: [
+                [
+                  sequelize.literal(`(
+                    SELECT COUNT(*)
+                    FROM "Members" AS "collective->members"
```

Comment:
I'm a bit afraid of the performance impact of summing members and transactions in these subqueries. Could we measure the complexity for `opensource`, `opencollective` and a smaller host to see how it performs?

Reply:
Also sharing something I've been discussing with @znarf, I think we all see the need for something like that: for this kind of use case, it would be great to have a `popularity` score somewhere (based on the contributor's count, and eventually the amount of money received) that would be re-computed by a CRON job every hour.

That would also be a valuable data to improve search.